plugins/modules/zabbix_screen.py

```diff
@@ -355,65 +343,43 @@ def create_screen_items(self, screen_id, hosts, graph_name_list, width, height,
                                                           'colspan': 1, 'rowspan': 1,
                                                           'elements': 0, 'valign': 0, 'halign': 0,
                                                           'style': 0, 'dynamic': 0, 'sort_triggers': 0})
-        except Already_Exists:
-            pass
+        except Exception as ex:
+            if ex.__class__.__name__ == "Already_Exists":
+                pass
+            else:
+                raise ex
```

Comment:
This is not very fortunate approach I would say. But on the other hand, this whole module could use som more love in the future.

Reply:
I know ... I don't like it too much either. But this way we could dump the imports for zabbix-api.
After this I would look into how we can use the Wrapper class to get around this nasty.